index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head> 
+<title>Portfolio</title>
+<link rel="stylesheet" href="styles.css">
+</head>
+<nav id="navbar">
+  <ul id="nav-list">
+    <li>
+      <a href="#welcome-section">About</a>
+    </li>
+    <li>
+      <a href="#projects">Work</a>
+    </li>
+    <li>
+      <a href="#contact">Contact</a>
+    </li>
+  </ul>
+</nav>
```

Comment:
Well done 👍   but I think that you shouldn't put the "nav" element between the "head" tag, but it should be between the "body" tag under a "header" tag. Even though there is no rule that strictly requires you to do this, doing this will keep the structure and hierarchy of HTML elements logical and semantically meaningful. 🎉